src/core/etl/src/Flow/ETL/Row/Entry/JsonEntry.php

```diff
@@ -11,28 +11,31 @@
 use Flow\ETL\Row\Schema\Definition;
 
 /**
- * @implements Entry<string, array{name: string, value: array<mixed>, object: boolean}>
+ * @implements Entry<string, array{name: string, value: array, object: boolean}>
  */
 final class JsonEntry implements \Stringable, Entry
 {
     use EntryRef;
 
-    private bool $object;
+    private bool $object = false;
+
+    private readonly array $value;
 
     /**
-     * JsonEntry constructor.
-     *
-     * @param array<mixed> $value
-     *
      * @throws InvalidArgumentException
      */
-    public function __construct(private readonly string $name, private readonly array $value)
+    public function __construct(private readonly string $name, array|string $value)
```

Comment:
I think that `fromJsonString` can also be now safely removed as it has no purpose with constructor accepting union type